README.md

```diff
@@ -0,0 +1,80 @@
+# Docker container for hubPredEvalsData generation
+
+This docker container is a wrapper around
+`hubPredEvalsData::generate_evals_data()` and hosts the in-development code
+from
+[hubverse-org/hubPredEvalsData](https://github.com/hubverse-org/hubPredEvalsData),
+which is used to generate tables of evaluation data from a hub's [oracle
+output](https://hubverse.io/en/latest/user-guide/target-data.html#oracle-output).
+
+The image is built and deployed to the GitHub Container Registry (https://ghcr.io).
+You can find the [latest version of the
+image](https://github.com/hubverse-org/hubPredEvalsData-docker/pkgs/container/hubpredevalsdata-docker/340871974?tag=main)
+by using the `main` tag:
+
+From the command line:
+
+```sh
+docker pull ghcr.io/hubverse-org/hubpredevalsdata-docker:main
+```
+
+## Usage
+
+The main usage for this image is a step in [the hub dashboard control
+room](https://github.com/hubverse-org/hub-dashboard-control-room) that builds
+evals data if it exists.
+
+The container packages the `create-predevals-data.R` script, which will display
+help documentation if you pass `--help` to it.
+
+```sh
+docker run --rm -it \
+-v "$(pwd)":/project \
+ghcr.io/hubverse-org/hubpredevalsdata-docker:main
```

Comment:
This is the only actual suggestion in this review: needed to add this `\` to get the R script to run.
```suggestion
ghcr.io/hubverse-org/hubpredevalsdata-docker:main \
```